msteams-platform/tabs/design/tabs.md

```diff
@@ -224,7 +226,7 @@ If you don’t use Teams color tokens, your designs will be less scalable and ta
 
 ## Validate your design
 
-If you plan to publish your app to AppSource, you should understand the design issues that commonly cause apps to fail during submission.
+If you plan to publish your app to the Teams store, you should understand the design issues that commonly cause apps to fail during submission.
```

Comment:
More content to help separate design guidelines (quality) from validation guidelines. Does this work?